python/custreamz/custreamz/deployment/custreamz_local/cuStreamz_job.py

```diff
@@ -0,0 +1,54 @@
+from distributed import Client
+#This creates a Dask client that submits jobs to the Dask scheduler that then assigns tasks to the workers.  
+client = Client("localhost:8786")
+client.get_versions(check=True)
+client
+
+###### Please write your cuStreamz job below ####
```

Comment:
These 3 files serve more as a simple pipeline that one can deploy once he has the cuStreamz setup. I feel it would be more convenient if these files were in the same folder as the deployment. If you still feel they should move into another folder, I can create a separate folder ' custreamz deployment' in notebooks-extended. This would be different from the example notebooks folder that would have something extremely simple like streaming_word_count.